ci-scripts/copy_to_archive

```diff
@@ -33,42 +37,53 @@ fi
 
 # Copy the logs
 if [ -d ${LOG_DIR}/isafw-logs ]; then
-	echo "Archiving logs to $ARCHIVE_DIR"
-	mv ${LOG_DIR}/isafw-report_*/* $ARCHIVE_DIR
+    echo "Archiving logs to $ARCHIVE_DIR"
+    mv ${LOG_DIR}/isafw-report_*/* $ARCHIVE_DIR
+fi
+
+if [ -f ${LOG_DIR}/cooker/**/console-latest.log ]; then
+    echo "Archiving console-latest.log file to $ARCHIVE_DIR"
+    cp -r ${LOG_DIR}/cooker/**/console-latest.log $ARCHIVE_DIR
+fi
+
+# Find the older directory
+LICENSE_BUILD_DIR=$(ls -t ${LICENSE_DIR} | grep "${IMAGE_BASENAME}.*20.*" | tail -1)
+# Copy license manifest file
+if [ -f ${LICENSE_DIR}/${LICENSE_BUILD_DIR}/license.manifest ]; then
+    echo "Archiving license.manifest file to $ARCHIVE_DIR"
+    mv ${LICENSE_DIR}/${LICENSE_BUILD_DIR}/license.manifest $ARCHIVE_DIR
 fi
 
-# If there is an Intel directory
-if [ -d ${IMAGE_DIR}/${INTEL_ARCH} ]; then
-    echo "Archiving Intel image to $ARCHIVE_DIR"
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+# Copy if image files exist
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic ]; then
+    echo "Archiving ${IMAGE_BASENAME}-rootfs.wic file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
 fi
 
-# If there is an ARP directory
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic.bmap ]; then
+    echo "Archiving ${IMAGE_BASENAME}-rootfs.wic.bmap file to $ARCHIVE_DIR"
```

Comment:
Consider rephrase to `Archiving rootfs.wic.bmap for ${IMAGE_BASENAME} to $ARCHIVE_DIR`